src/api/Jalasoft.TeamUp.Projects.API/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think we don't want to commit this file?

Reply:
I think It is necessary for launch in port 7072